modules/apps/31-ibc-query/types/msgs.go

```diff
@@ -28,14 +28,14 @@ func (msg MsgSubmitCrossChainQuery) GetQueryId() string { return msg.Id }
 
 func (msg MsgSubmitCrossChainQuery) GetPath() string { return msg.Path }
 
-func (msg MsgSubmitCrossChainQuery) GetTimeoutHeight() clienttypes.Height { return msg.LocalTimeoutHeight }
+func (msg MsgSubmitCrossChainQuery) GetTimeoutHeight() *clienttypes.Height {
```

Comment:
I don't think we need a pointer variable.
How about adding  `[(gogoproto.nullable) = false]` to height field in proto file ?

Reply:
`[(gogoproto.moretags) = "yaml:\"timeout_height\"", (gogoproto.nullable) = false];` 
If you add this, it will work properly. Please try again I'm sorry.